docs/main/vs-code-ext/1-new-project.md

```diff
@@ -0,0 +1,24 @@
+---
+title: New Project
+description: Visual Studio Code Extension for Capacitor
+contributors:
+  - dtarnawsky
+slug: /vscode/new-project
+---
+
+Follow these steps to create a **new starter project** for Angular, React or Vue. 
+
+<video style={{borderRadius:5}} width="100%" controls>
+      <source src={require('@site/static/video/new-project.mp4').default} type="video/mp4" />
+</video>
+
+1. Create a folder for your project and open it in VS Code
+2. Click the `Ionic` logo on the side menu
```

Comment:
Maybe consider standardizing on how to refer to the extension icon in the side menu?

`Web to Native` refers to it as:
>Click the Ionic extension on the side menu

while this page calls it a logo.